src/inode.rs

```diff
@@ -264,3 +251,27 @@ impl Inode {
         }
     }
 }
+
+fn get_inode_start_byte(ext4: &Ext4, inode: InodeIndex) -> Option<u64> {
+    let sb = &ext4.0.superblock;
+
+    // OK to unwrap: `inode` is nonzero.
+    let inode_minus_1 = inode.get().checked_sub(1).unwrap();
+
+    let block_group_index = inode_minus_1 / sb.inodes_per_block_group;
```

Comment:
I think a comment here isn't needed, because once all these fixups are done I'm going to enable `!#[warn(clippy::arithmetic_side_effects)]`. Any places with unchecked arithmetic will then produce a warning. No warning will occur here, since division by a `NonZero` type can't fail.